.travis.yml

```diff
@@ -1,11 +1,16 @@
 language: go
+sudo: required
+services:
+  - docker
 matrix:
   include:
-  - go: 1.10.3
+  - go: 1.x
```

Comment:
would recommend pinning to a specific version. I believe we pinned to `1.10.3` before because `1.11` was breaking everything at the time and we could not submit any other PR's before resolving that issue

Reply:
I'd swear I took 1.x from another kubernetes-csi repo, but I can't find it now. Restored 1.10.3.